src/main.rs

```diff
@@ -133,34 +193,21 @@ async fn listen_blocks(mut stream: mpsc::Receiver<near_indexer::BlockResponse>)
             })
             .collect::<Vec<String>>();
         update_receipt_status(succeeded_receipt_ids, StatusEnum::Success, &pool).await;
-
-        // Handle receipts
-        for chunk in &block.chunks {
-            diesel::insert_into(schema::access_keys::table)
-                .values(
-                    chunk
-                        .receipts
-                        .iter()
-                        .filter_map(|receipt| match receipt.receipt {
-                            near_indexer::near_primitives::views::ReceiptEnumView::Action {
-                                ..
-                            } => Some(AccessKey::from_receipt_view(
-                                receipt,
-                                block.block.header.height,
-                            )),
-                            _ => None,
-                        })
-                        .flatten()
-                        .collect::<Vec<AccessKey>>(),
-                )
-                .execute_async(&pool)
-                .await
-                .unwrap();
-        }
     }
 }
 
 fn main() {
+    let env_filter = EnvFilter::new("tokio_reactor=info,near=info,near=error,stats=info,telemetry=info,indexer_for_wallet=error,indexer_for_wallet=info");
```

Comment:
There is a duplication: `indexer_for_wallet=error,indexer_for_wallet=info`. Did you mean something else?

Reply:
No, I subscribe to `error` and to `info` for `indexer_for_wallet`

Reply:
The level is set to indicate the verbosity level, if you select INFO, it will include WARN and ERROR automatically. You override error setting with info setting here, so the former is useless.